components/script/parse/xml.rs

```diff
@@ -75,7 +77,13 @@ impl<'a> TreeSink for servoxmlparser::Sink {
 
     fn append(&mut self, parent: JS<Node>, child: NodeOrText<JS<Node>>) {
         let child = match child {
-            NodeOrText::AppendNode(n) => Root::from_ref(&*n),
+            NodeOrText::AppendNode(n) => {
+                let script = n.downcast::<HTMLScriptElement>();
```

Comment:
The HTML Sink impl handles execution of the javascript scripts. I added this here so javascript is executed when using xhtml; however, I am not sure if this would be the correct spot to do this.